src/main/java/medicaldoctor/controllers/DoctorDashboardController.java

```diff
@@ -0,0 +1,23 @@
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+/**
+ *
+ * @author Salmanhussain
```

Comment:
are we having the author note?

Reply:
Autogenerated.

Reply:
you can remove.
auotgenerated doesnt mean forced to be there